pages/learn/react/index.tsx

```diff
@@ -0,0 +1,16 @@
+import Head from 'next/head'
+
+import LearnReactContent from '@content/learn/react/index.mdx'
+
+export default function LearnReact() {
```

Comment:
instead of a raw mdx here we can use a React component and inject the mdx. That gives more control of the template and layout.